======= From backend =======
2024-01-22 12:14:18
Traceback (most recent call last):
  File "/opt/topspin3.6.5/exp/stan/nmr/py/user/poise_backend/backend.py", line 81, in main_wrapper
    main()
  File "/opt/topspin3.6.5/exp/stan/nmr/py/user/poise_backend/backend.py", line 145, in main
    x, y = getpar("F1P"), getpar("F2P")
  File "/opt/topspin3.6.5/exp/stan/nmr/py/user/poise_backend/cfhelpers.py", line 653, in getpar
    proc_f1 = _get_proc_par(par, p_spec / "proc2s")
  File "/opt/topspin3.6.5/exp/stan/nmr/py/user/poise_backend/cfhelpers.py", line 576, in _get_proc_par
    with open(p_procs, "r") as file:
FileNotFoundError: [Errno 2] No such file or directory: '/reference/ME/ubiCN/25/nhsqc.p90/1/pdata/999/proc2s'



======= From backend =======
2024-01-22 12:16:59
Traceback (most recent call last):
  File "/opt/topspin3.6.5/exp/stan/nmr/py/user/poise_backend/backend.py", line 81, in main_wrapper
    main()
  File "/opt/topspin3.6.5/exp/stan/nmr/py/user/poise_backend/backend.py", line 145, in main
    x, y = getpar("F1P"), getpar("F2P")
  File "/opt/topspin3.6.5/exp/stan/nmr/py/user/poise_backend/cfhelpers.py", line 653, in getpar
    proc_f1 = _get_proc_par(par, p_spec / "proc2s")
  File "/opt/topspin3.6.5/exp/stan/nmr/py/user/poise_backend/cfhelpers.py", line 576, in _get_proc_par
    with open(p_procs, "r") as file:
FileNotFoundError: [Errno 2] No such file or directory: '/reference/ME/ubiCN/25/nhsqc.p90/1/pdata/900/proc2s'



